### ISSUE TITLE
Get-WebFile doesn't properly handle authorization headers as options

### ISSUE THREAD

**Issue by steviecoaster** [chocolatey/choco#3738]
### What You Are Seeing?

Passing a basic authentication header to `Install-ChocolateyPackage` is not picked up appropriately.

### What is Expected?
`Install-ChocolateyPackage -Options $options` should be able to handle any kind of header

This appears to be a down-stream issue wherein `Get-ChocolateyWebFile` should consume those headers as part of the call, but it does not.

**Comment by TheCakeIsNaOH** [comment]
Duplicate of or related to #3191

**Comment by gep13** [comment]
@TheCakeIsNaOH I agree.  I will get this one marked as a duplicate, and close it out.  Thank you for noting this!

**Comment by gep13** [comment]
Duplicate of #3191
